aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SdkSettings.kt

```diff
@@ -20,4 +20,9 @@ class SdkSettings private constructor(private val awsSdk: ObjectNode?) {
     /** Path to AWS SDK integration tests */
     val integrationTestPath: String get() =
         awsSdk?.getStringMember("integrationTestPath")?.orNull()?.value ?: "aws/sdk/integration-tests"
+
+    /** Version number of the `aws-config` crate */
+    val awsConfigVersion: String get() =
+        awsSdk?.getStringMember("awsConfigVersion")?.orNull()?.value
+            ?: throw IllegalStateException("missing `awsConfigVersion` codegen setting")
```

Comment:
It uses a path dependency when it generates the service crate, so that should always be correct. The `fix-manifests` tool inserts version numbers after the fact. I think this all should be fine until we make runtime crates version independently, at which point this all needs to be revisited anyway. The mechanism for that revisiting will be the removal of the `smithy.rs.runtime.crate.version` Gradle property, which should reveal all the locations that rely on it, including this one.